website/docs/developer/deployment-guide/deployment-with-cumulus.md

```diff
@@ -385,6 +384,9 @@ of. More information on addressing workflow failures can be found on the
 ORCA [Best Practices](developer/../../development-guide/code/best-practices.mdx) 
 page.
 
+### Copy To Glacier lambda function
```

Comment:
Again, not really what I had in mind here and not really accurate either. I would remove this section and modify information in the workflow sections where we talk about copy_to_glacier including the workflow ASL information to better match what is needed.